node_modules/oae-util/lib/server.js

```diff
@@ -89,6 +89,13 @@ var setupServer = module.exports.setupServer = function(port, _config) {
         next();
     });
 
+    // Add CORS headers, cookies won't be passed in so all cross domain requests will be anonymous
+    app.use(function(req, res, next) {
+        res.header('Access-Control-Allow-Origin', '*');
+        res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
```

Comment:
I don't see any reason to, since the browser won't forward cookies; even non-GET requests will be anonymous. Also, I think this gives us better support for oauth authenticated requests (i.e. you could ouath and create content without proxying through a backend server now)